addons/binding/org.openhab.binding.windcentrale/README.md

```diff
@@ -0,0 +1,49 @@
+# WindCentraleBinding
+
+This Binding is used to display the details of a WindCentrale wind mill. 
+
+## Supported Things
+
+This Binding supports WindCentrale mills devices.
+
+## Discovery
+
+There is no discovery available for this binding.
+
+## Binding Configuration
+
+No binding configuration required.
+
+## Thing Configuration
+
+The thing mandatory configuration is the selection of the mill.
+Optional configuration is the number of Wind Parts ("Wind Delen") and the refresh interval.
+
+## Channels
+
+- **windSpeed** Measured current wind speed 
+- **windDirection** Current wind direction
+- **powerAbsTot** Total Power
+- **powerAbsWd** Power provided for your windparts
+- **powerRel** Relative power
+- **kwh** Current Energy
+- **kwhForecast** Energy Forecast
+- **runPercentage** Run Percentage this year
+- **timestamp** Timestamp of the last update
+
+
+## Full example
+```
+Group   gReiger "WindCentrale Reiger"   <wind>
+
+Number  ReigerWindSpeed         "Wind Snelheid[%1.0f Bf]"         <wind>    (gReiger) {channel="windcentrale:mill:reiger:windSpeed")
```

Comment:
I would doubt any non-Dutch would ever use this binding, hence I kept this in Dutch, so it can be easy pasted in any  item file.